packages/firestore/exp/src/api/components.ts

```diff
@@ -23,7 +23,7 @@ import {
   OfflineComponentProvider,
   OnlineComponentProvider
 } from '../../../src/core/component_provider';
-import {handleUserChange, LocalStore} from '../../../src/local/local_store';
+import { handleUserChange, LocalStore } from '../../../src/local/local_store';
```

Comment:
Looks like this PR includes some prettier fixes not caught during firestore or rules-unit-testing commits somehow, do you want to include them here or in a separate PR?